src/SearchResults.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = ({ results }) => {
+  return (
+    <table className="table">
+      <thead className="thead-dark">
+        <tr>
+          <th scope="col">id</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room id</th>
+          <th scope="col">Check in date</th>
+          <th scope="col">Check out date</th>
+          <th scope="col">nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {results.map(booking => {
+          return (
+            <tr>
+              <td>{booking.id}</td>
+              <td>{booking.title}</td>
+              <td>{booking.firstName}</td>
+              <td>{booking.surname}</td>
+              <td>{booking.email}</td>
+              <td>{booking.roomId}</td>
+              <td>{booking.checkInDate}</td>
+              <td>{booking.checkOutDate}</td>
+              <td>
+                {moment(booking.checkOutDate).diff(booking.checkInDate, "days")}
```

Comment:
Really good to see you making use of external libraries to solve problems, it's a good habit to get in to. A very small nitpick is that you can already do this in native JS with `checkoutDate.getDate() - checkInDate.getDate()`, as the javascript `Date` object allows things like subtracting/adding for dates.